src/components/Header/HeaderPrimary.vue

```diff
@@ -14,7 +14,6 @@ export default defineComponent({
   methods: {
     onResize() {
       this.mobileView = window.innerWidth <= 1024;
```

Comment:
So, we should definitely try to not handle window resize in JS and control the layout that way if possible.
(in your case it definitely is!)
In general, we should limit how much we control our layout directly by JS 😄
We should use media queries here and hide/show content that differs depending on the breakpoints.